core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -491,27 +524,47 @@ export class ContributionsAndReview
     this.SUGGESTION_TYPE_TRANSLATE = 'translate_content';
     this.TAB_TYPE_CONTRIBUTIONS = 'contributions';
     this.TAB_TYPE_REVIEWS = 'reviews';
+    this.TAB_TYPE_ACCOMPLISHMENTS = 'accomplishments';
     this.activeExplorationId = null;
     this.contributions = {};
     this.userDetailsLoading = true;
     this.userIsLoggedIn = false;
     this.activeTabType = '';
     this.activeSuggestionType = '';
     this.dropdownShown = false;
+    this.isAccomplishmentsTab = false;
     this.activeDropdownTabChoice = '';
     this.reviewTabs = [];
+    this.accomplismentsEnabled = (
+      this.featureService.status.ContributorDashboardAccomplishments.isEnabled);
     this.contributionTabs = [
```

Comment:
Is it possible to combine all these into a single constant rather than have two constants describing overlapping parts of the same thing? Or if not then maybe one constant for "order of tabs" and the other constant for "mapping tab names to tab properties"?